File: app/src/main/java/nosql/anew/chatmvp/utils/MyUtils.java
package nosql.anew.chatmvp.utils;

import java.text.SimpleDateFormat;
import java.util.Calendar;
import java.util.Date;
import java.util.TimeZone;

/**
 * Created by saksham on 25/6/17.
 */

public class MyUtils {
    public static String EXTRA_ROOM_NAME="EXTRA_ROOM_NAME";

    public static String convertTime(long timestamp) {
        Calendar c = Calendar.getInstance();
        SimpleDateFormat sdf;
        sdf = new SimpleDateFormat("HH:mm");
        Date date = new Date(timestamp);
        sdf.setTimeZone(TimeZone.getDefault());
        return sdf.format(date);
    }
}

File: Chatmvp - Kotlin/app/src/main/java/nosql/anew/kotlinmvp/utils/MyUtils.kt
package nosql.anew.kotlinmvp.utils

import java.text.SimpleDateFormat
import java.util.*

/**
 * Created by Sam on 11/04/18.
 */

object MyUtils {
    var EXTRA_ROOM_NAME = "EXTRA_ROOM_NAME"

    fun convertTime(timestamp: Long): String {
        val c = Calendar.getInstance()
        val sdf = SimpleDateFormat("HH:mm")
        val date = Date(timestamp)
        sdf.timeZone = TimeZone.getDefault()
        return sdf.format(date)
    }
}